cmd/agent/main.go

```diff
@@ -94,9 +94,14 @@ func setupZapLogger() zap.AtomicLevel {
 	}
 	cfg.EncoderConfig.EncodeTime = zapcore.RFC3339TimeEncoder
 	cfg.EncoderConfig.EncodeDuration = zapcore.StringDurationEncoder
+	ntpserv := os.Getenv("MA_NTP_SERVER")
+	if ntpserv == "" {
+		ntpserv = "pool.ntp.org"
+	}
+	ts := timesync.NewTimeSync(context.Background(), ntpserv, 0)
```

Comment:
```suggestion
	ts := timesync.NewTimeSync(context.Background(), global.AgentConf.Runtime.NTPServer, 0)
```

Reply:
Hi @iogakos , it seems that you didn't see last changes, all changes made to main.go are reverted to original because changes is made elsewere (git diff in this pr show changes but check on my repo, cmd/agent/main.go has no changes compared to your repo)

Reply:
@GuillaumeOuint Hm that's strange. For some reason if you look into the "Files changed" tab, this is not depicted. For example the changes in `main.go` are still on track to be added. But I also see your revert commit which is confusing.

Could you try squashing all commits into one and push again?